api/swagger.yaml

```diff
@@ -5428,6 +5428,35 @@ definitions:
         example:
           - "/etc/cdi"
           - "/var/run/cdi"
+      Containerd:
+        $ref: "#/definitions/ContainerdInfo"
+        x-nullable: true
+
+  ContainerdInfo:
+    description: |
+      Information for conntecting to the containerd instance that is used by the daemon.
```

Comment:
Didn't look at details yet, but noticed the typo here (`conntecting`)

Reply:
Fixed that :)